android/build.gradle

```diff
@@ -7,11 +7,11 @@ buildscript {
     def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['RNGH_kotlinVersion']
 
     repositories {
+        mavenCentral()
+        google()
         maven {
             url "https://plugins.gradle.org/m2/"
         }
```

Comment:
Why leave it here? mavenCentral + google is also enough

And in general, you can do as other packages do so that it does not affect the end user:
by example: 
https://github.com/invertase/react-native-firebase/blob/main/packages/app/android/build.gradle#L3